source/api_cc/src/DeepPot.cc

```diff
@@ -293,6 +294,7 @@ void DeepPot::init(const std::string& model,
   }
   cell_size = rcut;
   ntypes = get_scalar<int>("descrpt_attr/ntypes");
+  ntypes_spin = get_scalar<int>("descrpt_attr/ntypes_spin");
```

Comment:
This will break other models which do not have this tensor. You may need to use `try...catch`.